docs/roadmap.md

```diff
@@ -0,0 +1,27 @@
+# Roadmap
+
+The list is based on the survey results and the discussion in wg-etcd-operator. It is not set in stone and may be adjusted as needed.
+
+## v0.1.0
+- Create a new etcd cluster, e.g 3 or 5 members cluster
+  - Users should be able to set at least the etcd version and cluster size
+- Understand health of a cluster
+- Enabling TLS communication
+  - Should also support certificate renewal
+
+## v0.2.0
+- Upgrade across patches or one minor version
+- Scale in and out, e.g 3 members to 5 members or vise versa
+- Support customizing etcd flags
+
+## v0.3.0
+- Create on-demand backup of a cluster
+- Create periodic backup of a cluster
+- Create a new cluster from a backup
+
+## v0.4.0
+- Recover a single failed cluster member (still have quorum)
```

Comment:
I would move this before backup/restore. That is a separate and longer topic.

Reply:
done.